pkg/store/driver/tikv_driver.go

```diff
@@ -379,6 +429,15 @@ func (s *tikvStore) CurrentVersion(txnScope string) (kv.Version, error) {
 	return kv.NewVersion(ver), derr.ToTiDBErr(err)
 }
 
+// CurrentMinTimestamp returns current minimum timestamp across all keyspace groups.
+func (s *tikvStore) CurrentMinTimestamp() (uint64, error) {
+	ts, err := s.KVStore.CurrentMinTimestamp()
+	if err != nil && strings.Contains(err.Error(), "Unimplemented") {
```

Comment:
It's suggested to make it a special error and give it a type, instead of using string matching.